sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_helper.py

```diff
@@ -0,0 +1,15 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Let's create a folder **utils** folder,  a file called: **_utils.py** under the folder.

similar to: https://github.com/Azure/azure-sdk-for-python/blob/feature/communication-CallingServer-preview/sdk/communication/azure-communication-callingserver/tests/utils/_live_test_utils.py

Please create a class called class CallingServerUtils:
and make the method def is_valid_url(..) @staticmethod

Also, this method called by both sync and async api right? so the code convention should be  compatible with python 2.7, else it will fail in the ci-build